src/CoreBundle/Service/RequestInfoService.php

```diff
@@ -54,7 +54,10 @@ class RequestInfoService implements RequestInfoServiceInterface
      * @var bool
      */
     private $isCmsTemplateEngineEditModeCache;
-
+    /**
+     * @var bool
+     */
+    private $isPreviewModeCache;
```

Comment:
Actually, this is `bool|null`, not just `bool`

Reply:
fixed